app/views/shared/check_answers/_assets.html.erb

```diff
@@ -58,7 +63,7 @@
           <p>
             <%= govuk_link_to(
                   t("generic.change"),
-                  check_answer_url_for(journey_type, :partner_bank_accounts, @legal_aid_application),
+                  check_answer_url_for(journey_type, change_url, @legal_aid_application), # change this link truelayer/bank_statetment || non-passported
```

Comment:
Did you leave in this comment intentionally?

Reply:
Nope, that predated the `change_url` use! 🤦‍♂️ 
Good spot, will remove, rebase and move to UAT :+1: